Update section about code formatting (#1859)
mozilla/pontoon: CONTRIBUTING.rst

    iOS >= 10.3

Code style
==========

> We use code formatters so that we cannot fight over code style.

Reviewer: ```suggestion
We use code formatters so that we do not have to fight over code style.
```

> You are free to write code however you like, because in the end formatter is the one

Reviewer: ```suggestion
You are free to write code however you like, because in the end the formatter is the one
```

    $ make prettier 

Additioanally, there are linting rules that are defined in our
``.eslintrc.js`` file. To run the linter, do:

.. code-block:: shell

    $ make lint-frontend

In the rare case when you cannot fix an eslint error, use ``// eslint-disable`` to make the linter

> ignore that error. Note that in most cases, it is better to fix the issues than ignoring them.

Reviewer: ```suggestion
ignore that error. Note that in most cases, it is better to fix the issues than ignore them.
```
